cmd/api/src/model/filter.go

```diff
@@ -39,13 +39,15 @@ const (
 	LessThanOrEquals    FilterOperator = "lte"
 	Equals              FilterOperator = "eq"
 	NotEquals           FilterOperator = "neq"
+	ApproximatelyEquals FilterOperator = "~eq"
 
 	GreaterThanSymbol         string = ">"
 	GreaterThanOrEqualsSymbol string = ">="
 	LessThanSymbol            string = "<"
 	LessThanOrEqualsSymbol    string = "<="
 	EqualsSymbol              string = "="
 	NotEqualsSymbol           string = "<>"
+	ApproximatelyEqualSymbol  string = "ILIKE"
```

Comment:
ILIKE allows for case insensitive search

Reply:
ILIKE it!